content/api/commands/focus.md

```diff
@@ -4,6 +4,9 @@ title: focus
 
 Focus on a DOM element.
 
+It is [unsafe](/guides/retry-ability#Only-Queries-are-retried) to chain further
+commands that rely on the subject after `.focus()`.
```

Comment:
This contradicts our current recommendation for .blur() by saying this is unsafe to chain further...

> This element must currently be in focus. If you want to ensure an element is
focused before blurring, try using [`.focus()`](/api/commands/focus) before
`.blur()`.

Reply:
Updated the recommendation in `.blur()` to make it clearer.

> This element must currently be in focus. If you want to ensure an element is
focused before blurring, try using [`.focus()`](/api/commands/focus) before
`.blur()`.

> ```javascript
> cy.get('button').as('btn').focus()
> cy.get('@btn').blur()
> ```